src/loaders/express.js

```diff
@@ -34,6 +41,28 @@ module.exports = (app) => {
     app.use(passport.initialize());
     app.use(passport.session());
 
+    const mongo_uri = config.db_uri || `mongodb://${config.db_host}:${config.db_port}/${config.db_name}`;
```

Comment:
We have 2 separate definitions of this, even tough they are equal it would be best to have only one so if it ever needs be changed it would always stay consistent

Reply:
Made it so that the uri is always available if it is provided, or if host port and db name are provided, that way, one can always access db_uri directly. There were already checks to assert that there exists either db_uri or db host/port and name. Ran the tests and it successfully connected so it should be fine.

Reply:
The problem is not in connecting to the tests, as they provide db_host and db_port. The problem is that the new changes would probably not work in production, in which the mongodb cluster uri is provided :sweat_smile:

I think the suggested changes solve this though.